### ISSUE TITLE
Don't hide Tooltip on focus loss

### ISSUE THREAD

**Issue by zhrivodkin** [whoisandy/react-rangeslider#88]
Hi! Is there any way to don't hide the tooltip when it (or handle) loses focus?

**Comment by zhrivodkin** [comment]
#89

**Comment by kunal886496** [comment]
It is still not working i just tried . Using tooltip and keepTooltip both props

**Comment by HeratPatel1** [comment]
No, But You Can use "handleLabel" prop for that. Which provides the slider value on the slider dot.

![image](https://user-images.githubusercontent.com/36155316/36093077-7e8ff33e-0fb7-11e8-8d33-ddf24e8900a2.png)

[example](https://whoisandy.github.io/react-rangeslider/)
